wrappers/C#_High_Level_Wrapper/README.md

```diff
@@ -0,0 +1,48 @@
+# Wrapper of REFPROP for C&#35;
+
+The files in this directory demonstrate how to call REFPROP routines from a C# program for High level API. This example can be used to easily and quickly create your own C# program that uses RefProp.
+Cancel changes
+## License
+
+These files fall under a MIT-style license.
+
+## To Use Solution
+
+1. Do a git clone of the entire repository.
+2. Go into ``wrappers/Csharp`` directory
+3. Open the Visual Studio 2015 or later solution.
+
+## To Use Wraper
+
+1. Donwload the DLL wrapper : [wrappers/C#_High_Level_Wrapper/RefpropDLL/bin/Debug](https://github.com/meyinyil/REFPROP-wrappers/tree/master/wrappers/C%23_High_Level_Wrapper/RefpropDLL/bin/Debug)
```

Comment:
Should not point to your own repo (dangerous from a security standpoint). Better to provide instructions how to build yourself. Also, default to Release build.

Reply:
The link is deleted. Thanks for the comment.